packages/gradient-block/src/components/ColorTooltip.tsx

```diff
@@ -0,0 +1,52 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { joinClassNames, toHexString } from '@frontify/guideline-blocks-shared';
+import { EditAndDeleteColorBox } from './';
+import { ColorTooltipProps } from '../types';
+import { useHover } from '@react-aria/interactions';
+
+export const ColorTooltip = ({
+    gradientColor,
+    gradientColors,
+    showColorModal,
+    setColors,
+    setShowColorModal,
+    setCurrentlyEditingPosition,
+}: ColorTooltipProps) => {
+    const left = gradientColor.position > 95 ? `calc(${gradientColor.position}% - 8px)` : `${gradientColor.position}%`;
+    const { hoverProps, isHovered } = useHover({});
+
+    const pointClasses = joinClassNames([
+        'tw-w-2 tw-h-2 tw-rounded-full tw- tw-bg-line-x-strong -tw-mt-[22px]',
+        isHovered ? 'tw-bg-black-100' : 'tw-bg-black-20',
+    ]);
+
+    return (
+        <div className="tw-absolute tw-z-10" style={{ left }} {...hoverProps}>
+            <div data-test-id="color-points" className={pointClasses} />
+            {isHovered ? (
+                <div data-test-id="color-tooltip" role="tooltip" className="tw-inline-block tw-mt-2.5 -tw-ml-1">
```

Comment:
I would maybe just set the opacity to 0 instead of removing it from the DOM if it is not hovered. The reason being is currently it is a bit difficult to hover over the small dot and then move your cursor carefully downwards to edit the color.

Reply:
adjusted the handling its easier to use now